src/main/java/nextstep/ladder/User.java

```diff
@@ -0,0 +1,22 @@
+package nextstep.ladder;
+
+public class User {
+    private final String userName;
+    private final int MAX_USER_NAME_LENGTH = 5;
```

Comment:
정적 멤버 변수로 만들어 주는거 어떨까요? 😄 
```suggestion
    private static final int MAX_USER_NAME_LENGTH = 5;
```